.github/workflows/build-graphscope-dev-images.yml

```diff
@@ -41,9 +41,13 @@ jobs:
         VINEYARD_VERSION=${{ github.event.inputs.v6d_version }}
         if [[ -n ${VINEYARD_VERSION} ]]; then
           make dev-wheel VINEYARD_VERSION=${VINEYARD_VERSION}
+          make graphscope-dev VINEYARD_VERSION=${VINEYARD_VERSION}
+          make vineyard-dev VINEYARD_VERSION=${VINEYARD_VERSION}
         else
           # pull_request: use default vineyard_version
           make dev-wheel
+          make graphscope-dev
+          make vineyard-dev
```

Comment:
Also need to `make vineyard-runtime`, see [this](https://github.com/alibaba/GraphScope/blob/main/k8s/Makefile#L81).